lavamoat/browserify/beta/policy.json

```diff
@@ -4529,60 +4507,6 @@
         "define": true
       }
     },
-    "eth-lattice-keyring>gridplus-sdk>eth-eip712-util-browser": {
-      "globals": {
-        "intToBuffer": true
-      },
-      "packages": {
-        "eth-lattice-keyring>gridplus-sdk>eth-eip712-util-browser>bn.js": true,
-        "eth-lattice-keyring>gridplus-sdk>eth-eip712-util-browser>buffer": true,
-        "ethers>@ethersproject/keccak256>js-sha3": true
-      }
-    },
-    "eth-lattice-keyring>gridplus-sdk>eth-eip712-util-browser>bn.js": {
-      "globals": {
-        "Buffer": true
-      },
-      "packages": {
```

Comment:
This seems kinda strange to me why would a storybook update have anything to do with eth-lattice-keyring packages. @mcmire I think I need haaaaalp 😅

Reply:
@georgewrmarshall The `gridplus-sdk` package got updated very recently and this removed some sub-dependencies and changed a bunch of stuff in the LavaMoat policy files: https://github.com/MetaMask/metamask-extension/pull/15711. It seems like you might not be running `yarn lavamoat:auto` with the most up-to-date code.  Try merging `develop` and running `yarn lavamoat:auto` one more time. Sorry, I know this is painful 😞

Reply:
Ahh I see my issue I think it's because I had policy updates in this commit https://github.com/MetaMask/metamask-extension/pull/15684/commits/35c3e60fa1564687d101d5373fdbc97f69916704 I thought I had removed them in a rebase and then run `yarn lavamoat:auto` on just the storybook updates and the policy-overrides. Thanks for being so helpful and patient @mcmire I really appreciate it 🙏